tests/func/test_get.py

```diff
@@ -36,6 +36,28 @@ def test_get_repo_dir(repo_dir, erepo):
     trees_equal(src, dst)
 
 
+def test_get_regular_file(repo_dir, erepo):
+    src = erepo.REGULAR_FILE
+    dst = erepo.REGULAR_FILE + "_imported"
```

Comment:
I'm not against of changing erepo or creating a new fixture on top because we will soon have to test the same stuff for `dvc import`. But it is also okay to do that when we actually need it and for now just create files in-place as noted in https://github.com/iterative/dvc/pull/2837#discussion_r349930343

Reply:
I would prefer a separate fixture even in the future.